src/lib/Containers/Conversation.tsx

```diff
@@ -173,10 +173,15 @@ export default createFragmentContainer(Conversation, {
     fragment Conversation_me on Me {
       conversation(id: $conversationID) {
         id
+        __id
```

Comment:
This component issues the mutations, so it does need these new fields in the fragment.